examples/wallet/index.ts

```diff
@@ -107,8 +108,8 @@ if (!receiverPrivateKey) {
 
     // send from base account to receiver
     const txId = await web3Client.wallet().sendTransaction({
-      amount: '1',
-      fee: 0,
+      amount: fromMAS(1),
+      fee: fromMAS(0),
```

Comment:
why not 0n or 0? same line 159?
```suggestion
      fee: 0n,
```

Reply:
sure, I have changed it. Thakks